relnotes/verify.feature.md

```diff
@@ -0,0 +1,9 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to [Sociomantic assert/enforce
+  policies](https://github.com/sociomantic-tsunami/sociomantic/blob/master/Code/assert_vs_enforce.md)
```

Comment:
We'll need to adapt the guidelines to specifically address usage of assert vs enforce vs verify, but we can do that later.